docs/specs/tripgo.openapi.yaml

```diff
@@ -5036,6 +5036,53 @@ components:
           $ref: '#/components/schemas/ArrayOfSimpleDataSourceAttribution'
         localCost:
           $ref: '#/components/schemas/LocalCost'
+        geofences:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: Unique identifier of geofence
+              type:
+                type: string
+                description: Type of geofence
+                enum:
+                - CIRCLE
+              trigger:
+                type: string
+                description: When to trigger geofence
+                enum:
+                - ENTER
+                - EXIT
+              center:
+                description: Center coordinate for circular geofences
+                $ref: '#/components/schemas/Cooordinate'
```

Comment:
The coordinate schema seems to be missing @nighthawk

Reply:
pushed a temporary fix https://github.com/skedgo/tripgo-api/commit/aa3c02d982a81f871d0108a737101a04b6cff123, but it is probably not accurate